de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/ui/browser/BrowserAction.java

```diff
@@ -0,0 +1,35 @@
+package de.fu_berlin.inf.dpp.whiteboard.ui.browser;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public class BrowserAction {
+
+    public class ActionTypes {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=250&issues=AWY_InU0nzfF1D4JrUtY&open=AWY_InU0nzfF1D4JrUtY)

[ISSUE_KEY]: AWY_InU0nzfF1D4JrUtY